### ISSUE TITLE
Bot AI Captain issue for Radiant with multiple human players

### ISSUE THREAD

**Issue by evittalex** [Yewchi/vulft#9]
BUG REPORT : The new patch seems like a great upgrade. However in a radiant game with 3 human players, the teammate AI are not getting a "captain" set and thus just sit in the fountain without any further AI activity. We tried a wide arrangement of lobby slot selections and it still did not work. 

On dire side with 3 human players in Lobby slot 1, 3, and 4 and it worked fine to set a captain and behave normally.  However only one player seems able to set their lane and the rest are left to go where the bots did not.

**Comment by evittalex** [comment]
The dire side was a fluke so far. We can't recreate why it worked in that Lobby set up.  We then tried 1 player on Radiant, Lobby slot 1, and two players on Dire. Both team captains then worked?

We tried many different configurations in the lobby slots and are having a hard time coming up with any consistent success to get captains with 3 players on one team other than the one time listed in the previous comment.
